components/TheHeader.vue

```diff
@@ -34,23 +46,41 @@ export default {
   },
   data() {
     return {
-      searchTask: '',
+      search: false,
+      searchText: '',
+      debounced: null,
     };
   },
   computed: {
     ...mapGetters({
       locales: 'lang/getLocals',
       currentLocale: 'lang/getCurrentLocale',
+      isSearching: 'todos/getIsSearching',
     }),
   },
   mounted() {
     this.$i18n.setLocale(this.currentLocale.code);
+    this.debounced = debounce(this.searchTask, 2000);
```

Comment:
2s is a bit long. Let's wait half a second.

```suggestion
    this.debounced = debounce(this.searchTask, 500);
```

Reply:
Resolved